.github/workflows/master.yml

```diff
@@ -0,0 +1,51 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.3', '7.4']
+        phpunit-versions: ['latest']
```

Comment:
Hi @picandocodigo, just a heads up here about PHPUnit versions. I don't know how gh workflows works exactly but if with 'latest' here it grabs the latest PHPUnit compatible with specified PHP version, this might not work. [Latest WP supports PHPUnit up to version 7](https://make.wordpress.org/core/handbook/references/phpunit-compatibility-and-wordpress-versions/) but [PHPUnit's latest version that officialy supports PHP >= 7.3 is version 9](https://phpunit.de/supported-versions.html).

This is why in travis.yml I had to manually grab PHPUnit 7 for PHP >= 7.2 because otherwise Travis would grab PHPUnit 8 or 9 and the tests would fail.
https://github.com/picandocodigo/List-Category-Posts/blob/3e6bffc0b4f09e310cdbad2efb05a12ca4e97db4/.travis.yml#L27-L30

I tested it then and indeed it fails with newer versions.